### PR TITLE
Refs #28315 - FormField storybook

### PR DESCRIPTION
**Pull request by ezr-ondrej**

### CODE REVIEW

**Review by Ron-Lavi — COMMENTED** [review]
Thanks @ezr-ondrej !
could you fix the lint tests errors? looks related

File: webpack/assets/javascripts/react_app/components/common/forms/Form.stories.js [hunk 1]
```diff
@@ -78,14 +79,6 @@ storiesOf('Components/Form', module)
         <Row>
           <FormField {...ownComponentFieldProps} />
         </Row>
-        <Row>
-          <FormField
-            {...ownComponentFieldProps}
-            label="Customly rendered field"
-          >
-            <CustomSelectWithContext firstOption="CustomlyRenderedInput" />
```
Comment (MariaAga) [inline webpack/assets/javascripts/react_app/components/common/forms/Form.stories.js:86]: If this is deleted why keep the component definition?

Comment (ezr-ondrej) [reply webpack/assets/javascripts/react_app/components/common/forms/Form.stories.js:86]: You're right, thanks for noticing.

**Review by MariaAga — APPROVED** [review]

**Review by mmoll — APPROVED** [review]

**Comment by mmoll** [comment]
merged, díky @ezr-ondrej!

**Pull request merged**